models/Asset/MetaData/ClassDefinition/Data/Asset.php

```diff
@@ -16,18 +16,17 @@
 
 namespace Pimcore\Model\Asset\MetaData\ClassDefinition\Data;
 
-use Pimcore\Model\Element\ElementInterface;
 use Pimcore\Model\Element\Service;
 
 class Asset extends Data
 {
     public function normalize(mixed $value, array $params = []): mixed
```

Comment:
@blankse, I think we could also get rid of the `$params` parameter since it's not used .. (same for other methods).
WDYT?

Reply:
@Corepex It is part of the NormalizerInterface interface. This is also used for data object fields. There it is used for classificationstore for example.